src/main/java/com/iridium/iridiumskyblock/managers/IslandManager.java

```diff
@@ -808,6 +809,28 @@ public synchronized IslandMission getIslandMission(@NotNull Island island, @NotN
         }
     }
 
+    private String getDailyIslandMissions(@NotNull Island island, int index) {
+        List<String> islandMissions = IridiumSkyblock.getInstance().getDatabaseManager().getIslandMissionTableManager().getEntries(island).stream()
+                .filter(islandMission -> islandMission.getType() == Mission.MissionType.DAILY)
+                .map(IslandMission::getMissionName)
+                .distinct()
+                .collect(Collectors.toList());
+        if (islandMissions.size() > index) {
+            return islandMissions.get(index);
+        }
```

Comment:
We need more empty lines
```suggestion

        if (islandMissions.size() > index) {
            return islandMissions.get(index);
        }

```